Updated to reflect new Microsoft Flow capabilities.

KentWeareMSFT opened the pull request:
Microsoft Flow has evolved since this was originally created.  Updated to reflect current state of service.

azurecla commented:
Hi __@meet-bhagdev__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

meet-bhagdev commented:
meant to commit to azure content pr. This was a mistake

KentWeareMSFT committed:
Updated to reflect new Microsoft Flow capabilities.

Microsoft Flow has evolved since this was originally created.  Updated to reflect current state of service.
